indicators/NO_FUNC_001-004/R/NO_FUNC_001-004.qmd

```diff
@@ -0,0 +1,3139 @@
+
+### 2.3 Additional comments about the dataset
+
+none
+
+
+#### 2.3.1 Instructions for citing, using and accessing data
+
+ANO data are available from Kartkatalogen on the webpages of the Norwegian Environment Agency (as used here, https://kartkatalog.miljodirektoratet.no/Dataset/Details/2054) and via GBIF (https://doi.org/10.15468/4wc9r4).
+
+Reference species lists are available via https://artsdatabanken.no/Pages/281569/Generaliserte_artslistedatasett_til_artikkel_2.
+Plant indicator data from Grime and Tyler et al. are available via their references below. 
+
+## 3. Indicator properties
+
+### 3.1 Ecosystem Condition Typology Class (ECT)
+
+All functional plant community indicators reflect the plant species composition and thus represent ECT class 'B1 Biotic compositional characteristics'.
+Through the coupling of compositional data to environmental indicator values, the functional plant community indicators in addition represent the following ECT classes:
+Functional plant community indicators for nitrogen, phosphorus and pH reflect the availability of nutrients and soil chemical properties and thus represent the ECT class 'A2 Chemical characteristics'.
+Functional plant community indicators for heat requirement, moisture, light and abiotic stress (Grime's S) reflect the availability of abiotic, physical environmental resources and thus represent the ECT class 'A1 Physical characteristics'.
+Functional plant community indicators for grazing/mowing, soil disturbance, competition (Grime's C) and vegetation disturbance (Grime's R) reflect the availability of physical structuring conditions and thus represent the ECT class 'A1 Physical characteristics'.
+ 
+### 3.2 Ecosystem condition characteristic
+
+Functional plant community indicators in general are aimed at describing to which degree ecosystems fulfill the criterion 'Intact plant community', i.e. to that degree they are in good condition with respect to a plant community composition reflecting the functional properties of the respective ecosystem in the reference condition.
+
+The individual, specific functional plant community indicators represent concrete functional properties of an ecosystem and deviations from the reference thus shed light on which drivers affect the plant community.
+
+For instance, the functional plant community indicator on nitrogen represents the influence of nitrogen on the plant community composition in the given ecosystem. Thus, deviations from the reference community towards e.g. higher nitrogen values reflect an environment characterized by higher availability of nitrogen which has led to shifts in plant species composition towards higher abundances of species with higher affinity to nitrogen. This functional plant community indicator for nitrogen reflects both the chemical properties of the ecosystem but also the ecosystem’s productivity.
+
+### 3.3 Other standards
+
```

Comment:
Suggestion for ALIEN indicator:
In the Norwegian _Fagsystem for økologisk tilstand_ (ref Nybø and Evju....) the indicator is sorted under _Funksjonelt viktige arter og strukturer_.